packaging/fpm/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM alpine:3.8
```

Comment:
Since this is just for build tooling, it'd be nice if this image could be pushed to GitHub and built as part of the CI pipeline. WDYT about the effort to do that? 

My concern is around how we might end up having a lot of `grafana/fpm` images so It'd be easier to scope it to the specific project we're using FPM for.

Reply:
Yeah it is mostly for build tooling. I was personally tempted to put it in the repo build-image since we need `go` to avoid spurious errors in the Makefile. The `fpm` image `package.sh` entrypoint basically does the same thing as the `build-image` too. The only thing is that you are re-using the Loki build-image right now so I'd need to make that change there.

Reply:
I wouldn't mind forking the Loki build-image and adding FPM in the Agent fork of it; I'm far more comfortable with an `agent-build-image` on Dockerhub than I am with something specifically for FPM